### PR TITLE
Fix some issues

### LINKED ISSUE #377: Display additional units
ex: https://beta.opendota.com/matches/2743155083

### PR DESCRIPTION
**Pull request by blukai**
Fixes #377

@howardchung, I tried to find out for issue #387, check 2nd commit.

### CODE REVIEW

File: src/components/Match/matchColumns.jsx [hunk 1]
```diff
@@ -160,6 +160,7 @@ export const overviewColumns = match => [{
   field: 'items',
   displayFn: (row) => {
     const itemArray = [];
+    const itemBearArray = [];
```
Comment (howardchung) [inline src/components/Match/matchColumns.jsx]: It may not be bear only, even that's currently the case.

**Comment by howardchung** [comment]
We'll have to apply this new tooltip code to a few tooltips and perf test it.

File: src/components/Match/matchColumns.jsx [hunk 2]
```diff
@@ -169,8 +170,24 @@ export const overviewColumns = match => [{
           inflictorWithValue(itemKey, formatSeconds(firstPurchase))
         );
       }
+
+      if (row.hero_id === 80 && row.additional_units) {
```
Comment (blukai) [inline src/components/Match/matchColumns.jsx]: http://dev.dota2.com/showthread.php?t=132401

Comment (howardchung) [inline src/components/Match/matchColumns.jsx]: Gg valve

Comment (howardchung) [inline src/components/Match/matchColumns.jsx]: But put a comment explaining this for posterity

**Comment by blukai** [comment]
image

Currently applied for parsed status on players/overview and confirmed, registered, solo mmr on matches

**Comment by howardchung** [comment]
Can we do it for from now tooltip and KLA?

**Comment by blukai** [comment]
Sure, we can do it for all tooltips that doesn't require html 🍀

**Comment by blukai** [comment]
If this tooltip will work fine after some testing, we should replace with it table th tooltips and others.

**Comment by howardchung** [comment]
Will you do FromNowTooltip in this PR?

**Comment by blukai** [comment]
Oh, yes, I forget about it, will do it now.

**Comment by howardchung** [comment]
There seems to be a short delay before the tooltip shows up.  Is there some property we can override?

**Comment by blukai** [comment]
https://github.com/odota/ui/pull/389/files#diff-bab05dbdb7e778b54189549d47041d33R43

I did it because it can be annoying when user hover by chance over icons and it just appear tooltips. If he want to see it, small delay will not affect bad.

**Comment by howardchung** [comment]
I did some quick testing with pages of 20 matches and it looks ok

**Comment by blukai** [comment]
@howardchung, If everything ok with current changes, u can merge this. Can't use new tooltip for table th since table container have `overflow: hidden`, maybe find out a bit letter with th.

**Pull request merged**